setup.py

```diff
@@ -66,7 +66,7 @@
         "Programming Language :: Python :: Implementation :: PyPy",
         "Topic :: Database :: Front-Ends",
     ],
-    install_requires=["click", "requests", "six"],
+    install_requires=["click", "requests", "six", "google_oauth2_tool"],
```

Comment:
And why do you prefer the `google_oauth2_tool` package? Are there any specific reasons? Is this the project URL (https://github.com/dlancer/google-oauth2-tool)?
I noticed that this is not an official GCP python client supported by Google (meaning it may break in the future) and hasn't been updated for more than 1 year.

Reply:
Switched to `google_oauth`. Thanks.